app/components/special-coverage/special-coverage-edit/special-coverage-edit.html

```diff
@@ -107,9 +107,8 @@
         <label for="specialCoverageCampaign">Sponsor Campaign</label>
         <autocomplete-basic
             class="form-control"
-            ng-if="!model.campaign || model.campaign.$resolved"
```

Comment:
Tests pass locally, but fail in Travis. Appears to be some karma timeouts.
Have you run into this before?

On Wed, Jan 27, 2016 at 11:36 AM, Andrew Kos <notifications@github.com>
wrote:

> In
> app/components/special-coverage/special-coverage-edit/special-coverage-edit.html
> <https://github.com/theonion/bulbs-cms/pull/96#discussion_r51019213>:
>
> > @@ -107,9 +107,8 @@
> >          <label for="specialCoverageCampaign">Sponsor Campaign</label>
> >          <autocomplete-basic
> >              class="form-control"
> > -            ng-if="!model.campaign || model.campaign.$resolved"
>
> I think, because of the way autocomplete-basic works at the moment,
> there's no way around the flicker.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theonion/bulbs-cms/pull/96/files#r51019213>.
>



-- 
Mike Parent
Developer
Onion, Inc.